src/java/org/apache/ivy/plugins/repository/ssh/SshCache.java

```diff
@@ -263,9 +263,8 @@ public void clearSession(Session session) {
      * @param session
      *            to connect to
      * @return channelSftp or null if not successful (channel not existent or dead)
-     * @throws IOException if something goes wrong
      */
-    public ChannelSftp getChannelSftp(Session session) throws IOException {
+    public ChannelSftp getChannelSftp(Session session) {
```

Comment:
Similar comment as here https://github.com/apache/ant-ivy/pull/52/files#r127395633 applies to this line